build/media_source/system/js/inline-editing.es6.js

```diff
@@ -0,0 +1,312 @@
+  const container = document.querySelector('.inline-editing-container');
+  const form = document.querySelector('.inline-editing-form');
+  let inputField = null;
+
+  const hideContainer = () => {
+    container.classList.add('d-none');
+  };
+  hideContainer();
+
+  const cancelButton = document.createElement('button');
+  const cancelButtonIcon = document.createElement('span');
+  const saveButton = document.createElement('button');
+  const saveButtonIcon = document.createElement('span');
+  const buttons = document.createElement('span');
+
+  // Save button
+  saveButtonIcon.classList.add('icon-check');
+  saveButtonIcon.setAttribute('aria-hidden', 'true');
+  saveButton.type = 'button';
+  saveButton.classList.add('btn-primary');
+  Object.assign(saveButton.style, {
+    margin: '2px 5px',
+  });
+  saveButton.appendChild(saveButtonIcon);
+
+  // Cancel button
+  cancelButtonIcon.classList.add('icon-times');
+  cancelButtonIcon.setAttribute('aria-hidden', 'true');
+  cancelButton.type = 'button';
+  cancelButton.classList.add('btn-danger');
+  Object.assign(cancelButton.style, {
+    margin: '2px 5px',
+  });
+  cancelButton.appendChild(cancelButtonIcon);
+
+  buttons.appendChild(saveButton);
+  buttons.appendChild(cancelButton);
+  Object.assign(buttons.style, {
+    display: 'flex',
+    'flex-direction': 'row',
+    'align-items': 'center',
+  });
+
+  const { icon } = Joomla.getOptions('inline-editing');
+  const cover = document.createElement('span');
+  const loader = document.createElement('img');
+
+  // Loading icon
+  loader.src = icon;
+  loader.width = 30;
+  loader.height = 30;
+  Object.assign(loader.style, {
+    position: 'absolute',
+    top: 0,
+    bottom: 0,
+    left: 0,
+    right: 0,
+    margin: 'auto',
+    'z-index': 2,
+  });
+  Object.assign(cover.style, {
+    position: 'absolute',
+    top: 0,
+    bottom: 0,
+    left: 0,
+    right: 0,
+    'z-index': 1,
+    'background-color': '#aad4f959',
+  });
+
+  const showLoader = () => {
+    loader.classList.remove('d-none');
+    cover.classList.remove('d-none');
+  };
+  const hideLoader = () => {
+    loader.classList.add('d-none');
+    cover.classList.add('d-none');
+  };
+
+  const appendLoader = (element, hidden = true) => {
+    element.appendChild(loader);
+    element.appendChild(cover);
+    if (hidden) {
+      hideLoader();
+    } else {
+      showLoader();
+    }
+  };
+
+  container.appendChild(buttons);
+  appendLoader(container);
+
+  const format = (fieldHtml) => {
+    const parser = new DOMParser();
+    const doc = parser.parseFromString(fieldHtml, 'text/html');
+    const updatedField = doc.querySelector('.controls').children[0];
+
+    switch (inputField.tagName) {
+      case 'SELECT': {
+        const selectedOptions = updatedField.querySelectorAll('option:checked');
+        const display = [];
+
+        [].forEach.call(selectedOptions, (el) => {
```

Comment:
@rs4231199 You can use the  following which reads easier.
```js
selectedOptions.forEach(option => {
```
Change this for all the other cases as well.